modules/oracle/src/test/scala/com/dimafeng/testcontainers/OracleSpec.scala

```diff
@@ -2,6 +2,7 @@ package com.dimafeng.testcontainers
 
 import java.sql.DriverManager
 
+import com.dimafeng.testcontainers.implicits._
```

Comment:
(I haven't used scala for a while so my memory on implicit resolution is not that fresh) Can we somehow workaround this import so the implicit conversion comes without explicit import (e.g. define the conversion method in the base class)?

Reply:
For custom type we could solve it by adding implicit conversion to companion object, but we have plain String here.
One solution would be to add `DockerImageNameConverters` trait to `TestContainersSuite` in scalatest and munit modules. That should allow switching to new version without any code changes in most cases. Is that acceptable?

Reply:
That should work. @LMnet do you have thoughts on this?

Reply:
That's exactly what I mean by saying: "Users should not export any orphaned implicits to make the new implicit conversion work."

I thought about adding this implicit conversion to all container companions, but the solution with `TestContainersSuite` is much better. Go for it @pbylicki

Reply:
Thanks, added.